portal/mailservice/mailer.py

```diff
@@ -0,0 +1,35 @@
+import smtplib
+
+from config import HOST, PORT, TLS, USER, PASSWORD, FROM, SEND_EMAILS
+
+
+class Mailer:
+    def __init__(self, send_emails=SEND_EMAILS):
+        self.__send_emails = send_emails
```

Comment:
Its a boolean which decides whether emails should be sent. This should be set to False when testing so that you dont send emails to random people.